src/librustc_span/lib.rs

```diff
@@ -1044,9 +1044,7 @@ impl Decodable for SourceFile {
                 name_was_remapped,
                 unmapped_path: None,
                 // `crate_of_origin` has to be set by the importer.
-                // This value matches up with `rustc_hir::def_id::INVALID_CRATE`.
```

Comment:
`INVALID_CRATE` appears to have been removed at some point. Figured an option would be more appropriate now.